PhysicsTools/PatAlgos/src/MuonMvaIDEstimator.cc

```diff
@@ -0,0 +1,108 @@
+#include "PhysicsTools/PatAlgos/interface/MuonMvaIDEstimator.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+//#include "FWCore/Framework/interface/stream/EDProducer.h"
+//#include "FWCore/Utilities/interface/StreamID.h"
+//#include "FWCore/Framework/interface/stream/EDAnalyzer.h"
+
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "CommonTools/MVAUtils/interface/GBRForestTools.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonSelectors.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/JetReco/interface/PFJetCollection.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "JetMETCorrections/JetCorrector/interface/JetCorrector.h"
+
+using namespace pat;
+using namespace cms::Ort;
+
+MuonMvaIDEstimator::MuonMvaIDEstimator(const edm::FileInPath &weightsfile) {
+  randomForest_ = std::make_unique<ONNXRuntime>(weightsfile.fullPath());
+  std::cout << randomForest_.get() << std::endl;
```

Comment:
```suggestion
  LogDebug("MuonMvaIDEstimator") << randomForest_.get();
```

Reply:
@andrea21z   I see above has not been addressed.  Please check and update.